nixos/modules/services/web-apps/onlyoffice.nix

```diff
@@ -222,6 +228,13 @@ in
             cp -r ${cfg.package}/etc/onlyoffice/documentserver/* /run/onlyoffice/config/
             chmod u+w /run/onlyoffice/config/default.json
 
+            #Allow LAN-range IPs if set true
+            ${lib.optionalString (cfg.allowLocalConnections) ''
+            sed -i 's/"allowPrivateIPAddress": false/"allowPrivateIPAddress": true/' /run/onlyoffice/config/default.json
+            sed -i 's/"allowMetaIPAddress": false/"allowMetaIPAddress": true/' /run/onlyoffice/config/default.json
+            ''}
+
+
```

Comment:
We don't want to use sed here if we have jq below.

Reply:
Thanks for encouraging me to use the right tool! Done.